crates/db_schema/src/source/post.rs

```diff
@@ -182,13 +182,21 @@ pub struct PostUpdateForm {
 #[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
 #[cfg_attr(
   feature = "full",
-  derive(Identifiable, Queryable, Selectable, Associations, TS,)
+  derive(
+    Identifiable,
+    Queryable,
+    Selectable,
+    Associations,
+    TS,
+    CursorKeysModule
+  )
 )]
 #[cfg_attr(feature = "full", diesel(belongs_to(crate::source::post::Post)))]
 #[cfg_attr(feature = "full", diesel(table_name = post_actions))]
 #[cfg_attr(feature = "full", diesel(primary_key(person_id, post_id)))]
 #[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
 #[cfg_attr(feature = "full", ts(export))]
+#[cfg_attr(feature = "full", cursor_keys_module(name = post_actions_keys))]
```

Comment:
What is this for? The library doesnt have any docs unfortunately.

Reply:
Its for sorting. I use this in 2 upcoming PRs, that are fairly simple and will make it clear.

Reply:
The `CursorKeysModule` derive here creates the `db_schema::source::post::post_action_keys` module. Until I add missing docs, you might understand it better by looking at references to the module's items. I couldn't quickly find a reference to `post_actions_keys`, but post view imports a similar module (`post_keys as key`).